documentation/quick_start_qemu/quick_start_qemu_linux.md

```diff
@@ -2,34 +2,74 @@
+
+> - Install the compiler. If the compiler version installed with `apt-get` command is too old, it will cause compilation errors. You can download and install the new version using the following command in turn. The download link and the decompression folder name will vary according to the download version. The following Compression Packet will unzip to the `/opt` folder.
+>
+>   - `wget https://armkeil.blob.core.windows.net/developer/Files/downloads/gnu-rm/6-2016q4/gcc-arm-none-eabi-6_2-2016q4-20161216-linux.tar.bz2`
+>   - `cd /opt`
+>   - `sudo tar xf ~/gcc-arm-none-eabi-6_2-2016q4-20161216-linux.tar.bz2`
+>
+>
+> - After the compiler is installed, it is necessary to modify the `rtconfig.py` file under `rt-thread/bsp/qemu-vexpress-a9` BSP, modify the corresponding path to the bin directory corresponding to the compiler decompressed into the opt directory. Referring to the following figure, the directory name varies according to the downloaded version of the compiler:
+>
+> ![edit EXEC_PATH in rtconfig.py](figures/ubuntu-rtconfig-py.png)
+>
+
 
-- Install Git on your PC : `sudo apt-get install git`
 
-- Download RT-Thread Source Code : `git clone https://github.com/RT-Thread/rt-thread.git`
+## 3 Build QEMU Project
 
-- Install QEMU : `sudo apt-get install qemu`
+### 3.1 Move into QEMU folder
 
-- Install Scons : `sudo apt-get install scons`
+```
+cd rt-thread/bsp/qemu-vexpress-a9/
+```
 
-- Install the compiler. If the compiler version installed with `apt-get` command is too old, it will cause compilation errors. You can download and install the new version using the following command in turn. The download link and the decompression folder name will vary according to the download version. The following Compression Packet will unzip to the `/opt` folder.
+### 3.2 Configure the environment of Env tool
 
-  - `wget https://armkeil.blob.core.windows.net/developer/Files/downloads/gnu-rm/6-2016q4/gcc-arm-none-eabi-6_2-2016q4-20161216-linux.tar.bz2`
-  - `cd /opt`
-  - `sudo tar xf ~/gcc-arm-none-eabi-6_2-2016q4-20161216-linux.tar.bz2`
+#### 3.2.1 Remap python command
 
-- Install ncurses Library : `sudo apt-get install libncurses5-dev`
+We need to remap `python` command as python3 by default.
 
+Using `whereis` command to identify your python3's version:
 
-- After the compiler is installed, it is necessary to modify the `rtconfig.py` file under `rt-thread/bsp/qemu-vexpress-a9` BSP, modify the corresponding path to the bin directory corresponding to the compiler decompressed into the opt directory. Referring to the following figure, the directory name varies according to the downloaded version of the compiler:
+![python3-version](figures/python3-version.png)
 
-![edit EXEC_PATH in rtconfig.py](figures/ubuntu-rtconfig-py.png)
+For instance, as you can see, in my computer, the python3's version is python 3.9. You need to identify python3's version in your computer. Then, we remap the `python` command as python3 by default:
```

Comment:
For instance, on this computer, python3's version is python 3.9. You need to identify python3's version on your computer. Then, we remap the `python` command as python3 by default: